blocks/route.ts

```diff
@@ -81,20 +81,15 @@ const mapHandlers = (
           .bind(resolver);
 
         const $live = await ctxResolver(key);
-        return val!(request, {
-          ...context,
-          state: {
-            ...context.state,
-            $live,
-            resolve: ctxResolver,
-          },
-        });
+        context.state = { ...context.state, $live, resolve: ctxResolver };
```

Comment:
I had to set the context.state so that I could modify the state for the downstream resolvers. Previously It wasn't possible because I was copying the reference.